ui/package.json

```diff
@@ -25,9 +25,13 @@
     "@testing-library/jest-dom": "^5.11.4",
     "@testing-library/react": "^11.1.0",
     "@types/jest": "^26.0.14",
+    "@types/leaflet": "^1.5.19",
```

Comment:
We can do this another PR, but we should move the `@types` dependencies to `devDependencies`